src/index.js

```diff
@@ -6,3 +6,10 @@ import registerServiceWorker from './registerServiceWorker';
 
 ReactDOM.render(<App />, document.getElementById('root'));
 registerServiceWorker();
+// root.render(
+//   <React.StrictMode>
+//     <App />
+//   </React.StrictMode>
+// );
+// Not sure if I need this code but it was in the live code, code gets sent into render method and once its rendered its injected
+// directly into html inside the anchor div
```

Comment:
You don't _need_ this code, and I verified that it works without it, but there _is_ an advantage to keeping it. Namely, it will render the `App` component in strict mode, which doesn't produce any UI, but _will_ produce warnings that may help you find bugs.

You can read more about it [in the React docs on Strict Mode](https://reactjs.org/docs/strict-mode.html).